apps/dashboard/src/lib/thirdweb-client.ts

```diff
@@ -1,12 +1,10 @@
-import { DASHBOARD_THIRDWEB_SECRET_KEY } from "@/constants/env";
-import { DASHBOARD_THIRDWEB_CLIENT_ID } from "constants/rpc";
+import {
```

Comment:
removing the duplication here makes sense but can we move it the *opposite* direction? I'd like to keep `@constants/client` and remove this file instead.

Reply:
maybe even `@/client`